timesketch/frontend/src/views/Explore.vue

```diff
@@ -21,37 +21,65 @@ limitations under the License.
       </template>
     </ts-navbar-main>
 
-    <ts-navbar-secondary currentAppContext="sketch" currentPage="explore"></ts-navbar-secondary>
+    <ts-navbar-secondary ref="navigation" currentAppContext="sketch" currentPage="explore"></ts-navbar-secondary>
 
     <section class="section">
       <div class="container is-fluid">
         <div class="card">
-          <div class="card-content" v-if="showSearch">
-            <div class="field has-addons">
-              <div class="control">
-                <ts-view-list-dropdown
-                  @setActiveView="searchView"
-                  @clearSearch="clearSearch"
-                  :current-query-string="currentQueryString"
-                  :current-query-filter="currentQueryFilter"
-                  :view-from-url="params.viewId"
-                  :sketch-id="sketchId"
-                ></ts-view-list-dropdown>
+          <b-modal :active.sync="showSaveSearchModal" :width="640" scroll="keep">
+            <div class="card">
+              <header class="card-header">
+                <p class="card-header-title">Save search</p>
+              </header>
+              <div class="card-content">
+                <div class="content">
+                  <ts-create-view-form
+                    @setActiveView="searchView"
+                    :sketchId="sketchId"
+                    :currentQueryString="currentQueryString"
+                    :currentQueryFilter="currentQueryFilter"
+                  ></ts-create-view-form>
+                </div>
               </div>
-              <div class="control" style="width: 100%;">
+            </div>
+          </b-modal>
+
+          <div class="card-content" v-if="showSearch">
+            <div style="position:relative;">
+              <div class="ts-search-box" style="z-index:999; position:absolute; width:100%;">
+                <span class="icon" style="position:absolute;top:14px;margin-left:17px;font-size:16px;">
+                  <i class="fas fa-search"></i>
+                </span>
                 <input
                   @keyup.enter="search"
                   v-model="currentQueryString"
+                  v-on:click="showSearchDropdown = true"
```

Comment:
Would you consider changing it to
`v-on:click="showSearchDropdown = !showSearchDropdown"`

So that clicking into the search bar the 2nd time would toggle the dropdown?

Reply:
Let's consider this. I was looking at how other big search dropdowns are handling this, and they all keep the dropdown open when clicking the input field, so I opted to follow that UX flow.